src/utils/utils.ts

```diff
@@ -41,3 +46,5 @@ export function untilDocumentReady() {
         document.addEventListener('readystatechange', resolve, { once: true, passive: true })
     })
 }
+
+export const nop: (...args: any[]) => any = () => {};
```

Comment:
```ts
export { noop } from 'lodash-es'
```

Reply:
noop, which has a type ```void```, is not a compatible with type ```() => void```